plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -183,11 +189,22 @@ static bool nvenc_update(void *data, obs_data_t *settings)
 		enc->context->rc_min_rate = bitrate * 1000;
 		cqp = 0;
 	}
-
+	
+	if (rclookahead > 0)
+	{
+		av_opt_set_int(enc->context->priv_data, "b_adapt", false, 0);
```

Comment:
You aren't exposing these options to the user.  They're just being forced to their non-default settings.  Why?